ClassPaulita/GuessTheCardGame.cs

```diff
@@ -109,22 +264,32 @@ private void Play()
 
                 if (userGuess == char.ToLower('y') && currentCard.CardValue > previousCard.CardValue || userGuess == char.ToLower('n') && currentCard.CardValue <= previousCard.CardValue)
                 {
-                    Console.WriteLine($"\n{spacing}Correct! The next card is {currentCard}.");
+                    Console.WriteLine($"\n{_spacing}Correct! The next card is {currentCard}.");
 
                     userScore++;
 
-                    Console.WriteLine($"{spacing}Your current score is: {userScore} point(s). Keep it up!");
+                    Console.WriteLine($"{_spacing}Your current score is: {userScore} point(s). Keep it up!");
                 }
 
                 else
                 {
-                    Console.WriteLine($"\n{spacing}Incorrect! The next card was {currentCard}.");
+                    Console.WriteLine($"\n{_spacing}Incorrect! The next card was {currentCard}.");
 
-                    Console.WriteLine($"{spacing}Your current score is: {userScore} point(s). Try to get the next one right!");
+                    Console.WriteLine($"{_spacing}Your current score is: {userScore} point(s). Try to get the next one right!");
                 }
 
                 previousCard = currentCard;
             }
+
+            if (userScore > _scoreBoard[4].Key)
+            {
+                AddUserScoreToBoard(userScore);
+            }
+
+            else
+                Console.WriteLine("Sorry! You did not make it to the LeaderBoard. Keep trying!");
```

Comment:
The rest of your single line if's and else's have brackets. Try to keep it consistent.

Reply:
picky.